Formula/coreutils.rb

```diff
@@ -69,9 +69,7 @@ def install
     # https://github.com/Homebrew/homebrew/issues/44993
     # This is thought to be an el_capitan bug:
     # https://lists.gnu.org/archive/html/bug-tar/2015-10/msg00017.html
-    if MacOS.version == :el_capitan
-      ENV["gl_cv_func_getcwd_abort_bug"] = "no"
-    end
+    ENV["gl_cv_func_getcwd_abort_bug"] = "no" if MacOS.version == :el_capitan
```

Comment:
@ilovezfs Formerly, yes, but RuboCop now handles it and has a specific line length that it applies it to. Currently this is 80 but can be shortened if desired.